### PR TITLE
Kc create folders for diff file types

### PR DESCRIPTION
**Pull request by kev1n80**
+ Moved all of the files into folder 
+ Added the changes in response to pr suggestions in this pr

### CODE REVIEW

File: src/main/webapp/templates/search.html [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+    <link rel="stylesheet" type="text/css" href="stylesheets/user.css">
+    <link rel="stylesheet" type="text/css" href="stylesheets/card.css">
+    <link rel="stylesheet" type="text/css" 
+        href="stylesheets/listing-preview.css">
+    <link rel="stylesheet" type="text/css" 
+        href="stylesheets/listing-detailed.css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <script type="module" src="miscellaneous.js"></script>
-    <script type="module" src="search.js"></script>
+    <script type="module" src="scrips/miscellaneous.js"></script>
```
Comment (zrideaux) [inline src/main/webapp/templates/search.html:13]: Change scrips to scripts

File: src/main/webapp/templates/user.html [hunk 2]
```diff
@@ -3,14 +3,16 @@
...
+    <link rel="stylesheet" type="text/css" href="stylesheets/user.css">
+    <link rel="stylesheet" type="text/css" href="stylesheets/card.css">
+    <link rel="stylesheet" type="text/css" 
+        href="stylesheets/listing-preview.css">
+    <link rel="stylesheet" type="text/css" 
+        href="stylesheets/listing-detailed.css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
         rel="stylesheet">
-    <script type="module" src="user.js"></script>
+    <script type="module" src="sripts/user.js"></script>
```
Comment (zrideaux) [inline src/main/webapp/templates/user.html:15]: Change sripts to scripts

File: src/main/webapp/templates/search.html (hunk 1 above)
Comment (kev1n80) [reply src/main/webapp/templates/search.html:13]: Nice catch!

File: src/main/webapp/search.html [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <script type="module" src="miscellaneous.js"></script>
-    <script type="module" src="search.js"></script>
+    <script type="module" src="scripts/miscellaneous.js"></script>
+    <script type="module" src="scripts/search.js"></script>
   </head>
   <body>
     <script type="module">
-      import { toggleDropdown } from './miscellaneous.js'
+      import { toggleDropdown } from './scripts/miscellaneous.js';
```
Comment (zrideaux) [inline src/main/webapp/search.html:18]: Small thing, but your import statements in this file (maybe others?) vary a bit!
Line 018 has curly braces and a dot before the rest of the path: `import { toggleDropdown } from './scripts/miscellaneous.js';`

Line 135 doesn't have these things: `import displayListings from "/scripts/search.js";`

Comment (kev1n80) [reply src/main/webapp/search.html:18]: Yup, depending on how I export the function in my js files determine how I import them. Usually I use export default when I am exporting only one function in my js file (which you would use line 135 to import it), but for js files with more than one function to export I will export using the curly braces (export {func1Name, func2Name, ...}) which I would use line 18 to import it.

**Pull request merged**